data/data/aws/bootstrap/main.tf

```diff
@@ -36,7 +40,7 @@ resource "aws_iam_instance_profile" "bootstrap" {
 }
 
 resource "aws_iam_role" "bootstrap" {
-  name = "${var.cluster_name}-bootstrap-role"
+  name = "${var.cluster_id}-bootstrap-role"
```

Comment:
While we're cleaning up, I think we can drop `-role`, which is already implied by the resource type.  Basically, every bootstrap resource can use `${var.cluster_id}-bootstrap` unless we create several of a single resource type.

Reply:
I think it is fine if we keep the names as is. it doesn't hurt here as `iam roles` do not seem to have any limitation on length of name.